library/alloc/src/string.rs

```diff
@@ -139,6 +139,12 @@ use crate::vec::Vec;
 /// [`bytes`]: str::bytes
 /// [`chars`]: str::chars
 ///
+/// You can, however, index into a `String` in constant-time with anything that
+/// implements [`SliceIndex<str>`]. See the documentation for [`get`] for more details.
+///
+/// [`SliceIndex<str>`]: core::slice::SliceIndex
```

Comment:
Not sure if we want to always link to `core` docs if the user was looking at, for instance, `std::string::String` I feel like we should be able to get them to `std::slice::SliceIndex` is that possible?

Reply:
I think that's not currently possible, so this is probably good as-is.